### DISCUSSION TITLE
Bunch of trained models are in fact merge. Merge should be the default value in the form.

### DISCUSSION THREAD

**Discussion by RayHell** [civitai/civitai#212] [Ideas]
There's way more merge than trained models and when you read the description most new trained models are in fact merge.
Solution: Either nothing is check by default or merge is set by default.

**Comment by JustMaier** [comment]
I'll see if we can make it so neither are selected by default. Would you mind sending me a few of the models that have been mis-labeled so that we can get them cleaned up?

**Comment by manuelurenah** [comment]
Hey @RayHell! We have already implemented a fix for this and should be out in our next release later today. Thanks again for pointing this out 🔥

**Comment by RayHell** [comment]
I've reported one, I've put a comment on others to make the user change the settings. Those are ok now.
Here's another one I just found.
https://civitai.com/models/6620/realistic-sci-model

That change will be helpful thank you for the implementation.
